src/devices/tuya.ts

```diff
@@ -7397,6 +7397,31 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+    fingerprint: [
+        {   modelID: 'TS0601',
+            manufacturerName: '_TZE204_5cuocqty',
```

Comment:
This can be merged with the already existing `TS0601_dimmer_5`

Reply:
thank you @Koenkk for your fast reaction! 
Does this mean that I don't have to do anything for now?

Reply:
thanks @Koenkk . i just merged that change to the existing TS0601_dimmer_5.
I hope I did it right now...:)

Reply:
Looks good, will merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/7072